aait/backend/group-2B/Application/Contracts/Persistence/IInteractionRepository.cs

```diff
@@ -2,4 +2,8 @@
 
 namespace SocialSync.Application.Contracts.Persistence;
 
-public interface IInteractionRepository : IGenericRepository<Interaction> { }
+public interface IInteractionRepository : IGenericRepository<Interaction> {  
+     Task<List<Interaction>> GetAllCommentInteractions(int PostId);
+
+    Task<Interaction> likeUnlikeInteraction(Interaction interaction);
```

Comment:
use Pascal case for method names(this rule applies for all methods).

Reply:
I have resolved it.